ui-tests/src/test/java/org/openmrs/reference/RegisterPatientTest.java

```diff
@@ -60,6 +61,4 @@ public void registerPatient() throws InterruptedException {
         assertThat(dashboardPage.getPatientGivenName(), is(patient.givenName));
         assertThat(dashboardPage.getPatientFamilyName(), is(patient.familyName));
     }
-
```

Comment:
Remove these lines incase you dont need them for proper /openmrs javaconventions

Reply:
> Remove these lines incase you dont need them for proper /openmrs javaconventions

Thanks @sherrif10 for the review. You may need to take a look at the second commit with id `ff8904d`, it was already catered for within it.